test/integration/cli-args/info.stan

```diff
@@ -24,17 +33,23 @@ parameters {
   corr_matrix[15] q;
   cholesky_factor_cov[16] r;
   cholesky_factor_corr[17] s;
+  real y;
 }
 
 transformed parameters {
   matrix[14, 14] t = p;
   {
-    real ignored_tp = r;
+    real ignored_tp = log(r[0][0]);
   }
 }
 
 model {
-    real ignored_model = r;
+    real ignored_model = square(foo(square(r[0][0])));
+    l ~ dirichlet(c);
+    target += normal_lpdf(y| 0, 1);
+    target += normal_lcdf(y| 0, 1);
+    target += normal_lccdf(y| 0, 1);
```

Comment:
```suggestion
    target += normal_lccdf(y| 0, 1);
    target += std_normal_lupdf(y)
```

This one should be listed as `std_normal_lpdf` (target += lupdf is the same as using the sampling statement).

For this rename you can just call Utils.stdlib_distribution_name on all distributions.